examples/telemetry-stdout.graphql

```diff
@@ -0,0 +1,28 @@
+schema
+  @server(port: 8000, graphiql: true, hostname: "0.0.0.0")
+  @upstream(baseURL: "http://jsonplaceholder.typicode.com", httpCache: true)
+  @telemetry(export: {stdout: {pretty: true}}) {
```

Comment:
I find the `export` key kind of unnecessary. I think we should just have the following instead:

```
@telemetry(stdout: {pretty: true}) 
```

Let's also rename this to `@trace` instead of `@telemetry`. Reason being, "telemetry" is a much broader term encompassing components that are responsible for - transmission, monitoring, generating insights etc. None of those are part of the Tailcall package.

Reply:
We need to also add apollo tracing somewhere in the same configuration.